README.md

```diff
@@ -1,25 +1,65 @@
 # presentation-microservices-kotlin
-Presentation code and material for micro-services with kotlin
+
+Build microservices wth Kotlin + Spring Boot
+Presented at GDG Kotlin/Everywhere Hong Kong : A Day of Kotlin, 31st Aug 2019
+Slides can be found here
 
 [![master](https://img.shields.io/badge/travis-master-blue.svg)][travis-url][![TravisCI][travis-master-badge]][travis-url]
 
+## Introduction
+
+Two backend services are created in this project, `auth` and `app` respectively. A microservice-based architecture can be achieved locally with docker-compose or Docker. We have also deployed them on Oursky's k8s cluster.
+
 ## Get Started
+
 ##### Generate secret
+
 ```
 cd backend
 make secret
 ```
-##### Start servers
+
+##### Start servers with docker-compose
+
 ```
 cd backend
 docker-compose -p demo up
 ```
+
+##### With Docker
+
+```
+cd backend
+docker build -f Dockerfile.auth . -t demo-auth
+docker run --rm -it -p 8080:8080 -t demo-auth
+# OR
+docker run --rm -it -p 8080:8080 $(docker build -f Dockerfile.auth -q .)
+```
+
+#### Or with on our k8s cluster
```

Comment:
I don't think we need to encourage access to our demo server. We don't have effort to closely monitor it and we might shut it down any time (and bring it on when needed).

Reply:
ughh a shame, reasonable though